common/features/config/networks/static/reducer.ts

```diff
@@ -851,7 +852,33 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       initial: 1
     }
   }
-};
+},
+
+  ETI: {
+    id: 'ETI',
+    name: 'Etherinc',
+    unit: 'ETI',
+    chainId: 101,
+    isCustom: false,
+    color: '#3560bf',
+    blockExplorer: makeExplorer({
+      name: 'Etherinc Explorer',
+      origin: 'https://explorer.einc.io'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: ETI_DEFAULT,
```

Comment:
As I stated in the previous PR, please include links to the relevant pull request in third party repositories.  It saves time for those reviewing the PR.  Meanwhile, I see your trezor PR was merged three hours ago.  https://github.com/trezor/trezor-common/pull/301  :+1: